README.md

```diff
@@ -27,3 +27,13 @@ about the current recursive call.
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. You do not need to prove that the invariant is correct. Add
 your answer to this markdown file.
+
+
+Answer: 
+The invariant for this code is a - 1.  I used (a) as (I). I is usually the index of the array. The reason is that with each recursive call, this function will look back at the previous call, and it will look at the sum of the previous two elements, and that will dictate that it can keep going. In this case, I used a as i.
```

Comment:
You're still not telling me what the invariant means. How is i - 1 an invariant?